src/art/betiniakarandut/meta.json

```diff
@@ -1,5 +1,12 @@
 {
+<<<<<<< HEAD
```

Comment:
Please fix this file. That's not valid json.

Reply:
Thank you for the quick observation. I have changed the folder name from betiniakarandut to betiniakarandut01.
Had merge conflict issues before and forgot to resolve it in the meta.json foldrer too. PENDING ISSUES RESOLVED.